libxrdp/xrdp_sec.c

```diff
@@ -778,20 +778,20 @@ xrdp_sec_process_logon_info(struct xrdp_sec *self, struct stream *s)
         return 1;
     }
 
-    if (unicode_in(s, len_domain, self->rdp_layer->client_info.domain, 255) != 0)
+    if (unicode_in(s, len_domain, 512, self->rdp_layer->client_info.domain, 255) != 0)
```

Comment:
The function name explains how the arguments are used. I don't see why the values are 512 and 255 for this field. We can have one define. But even five defines would be better than no defines IMHO.